app/webpacker/components/Results/Rankings/RankingsTable.jsx

```diff
@@ -61,140 +65,18 @@ export default function RankingsTable({ filterState }) {
     select: (rankingsData) => mapRankingsData(rankingsData, show === 'by region'),
```

Comment:
I can't comment on the actual line of the function, but `mapRankingsData` should be moved to `utils`, like `augmentApiResults` is.